src/utils.rs

```diff
@@ -31,6 +31,7 @@ pub fn report_error(e: &super::Error) {
         red("  caused by:");
         eprint!(" {}", e);
     }
+    eprintln!();
```

Comment:
From what I can tell those `eprint`s above should just be `eprintln`s instead.

Reply:
Thank you for the review. This tool uses error-chain and I'm not familiar with the crate. So I'm not sure, but at least first error message has newline correctly.
Here is an error output got in my project:

<details><pre><code>
> cargo +nightly fuzz run input_text                                                                                                                                                                                                                  [OfficeDev.local][10/25 11:10]
       Fresh cc v1.0.46
       Fresh fs_extra v1.1.0
       Fresh cfg-if v0.1.10
       Fresh arc-swap v0.4.3
       Fresh unicode-width v0.1.6
       Fresh arbitrary v0.1.1
       Fresh libc v0.2.65
       Fresh getopts v0.2.21
       Fresh dirs-sys v0.3.4
       Fresh jemalloc-sys v0.3.2
       Fresh signal-hook-registry v1.1.1
       Fresh termios v0.3.1
       Fresh term_size v0.3.1
       Fresh dirs v2.0.2
       Fresh signal-hook v0.1.10
       Fresh jemallocator v0.3.2
       Fresh libfuzzer-sys v0.1.0 (https://github.com/rust-fuzz/libfuzzer-sys.git#4ad88ec5)
       Fresh term v0.6.1
       Fresh kiro-editor v0.3.5 (/Users/rhysd/Develop/github.com/rhysd/kiro-editor)
   Compiling kiro-editor-fuzz v0.0.0 (/Users/rhysd/Develop/github.com/rhysd/kiro-editor/fuzz)
     Running `rustc --edition=2018 --crate-name input_text fuzz_targets/input_text.rs --color always --crate-type bin --emit=dep-info,link -C debuginfo=2 -C metadata=1f46cebbe29cadde -C extra-filename=-1f46cebbe29cadde --out-dir /Users/rhysd/Develop/github.com/rhysd/kiro-editor/fuzz/target/x86_64-apple-darwin/debug/deps --target x86_64-apple-darwin -C incremental=/Users/rhysd/Develop/github.com/rhysd/kiro-editor/fuzz/target/x86_64-apple-darwin/debug/incremental -L dependency=/Users/rhysd/Develop/github.com/rhysd/kiro-editor/fuzz/target/x86_64-apple-darwin/debug/deps -L dependency=/Users/rhysd/Develop/github.com/rhysd/kiro-editor/fuzz/target/debug/deps --extern kiro_editor=/Users/rhysd/Develop/github.com/rhysd/kiro-editor/fuzz/target/x86_64-apple-darwin/debug/deps/libkiro_editor-06bdf11b37286e1e.rlib --extern libfuzzer_sys=/Users/rhysd/Develop/github.com/rhysd/kiro-editor/fuzz/target/x86_64-apple-darwin/debug/deps/liblibfuzzer_sys-0d3a1916dd5a6e3c.rlib --cfg fuzzing -Cpasses=sancov -Cllvm-args=-sanitizer-coverage-level=4 -Cllvm-args=-sanitizer-coverage-trace-compares -Cllvm-args=-sanitizer-coverage-inline-8bit-counters -Cllvm-args=-sanitizer-coverage-trace-geps -Cllvm-args=-sanitizer-coverage-prune-blocks=0 -Cllvm-args=-sanitizer-coverage-pc-table -Zsanitizer=address -L native=/Users/rhysd/Develop/github.com/rhysd/kiro-editor/fuzz/target/x86_64-apple-darwin/debug/build/jemalloc-sys-470dba03bcb05edc/out/build/lib -L native=/Users/rhysd/Develop/github.com/rhysd/kiro-editor/fuzz/target/x86_64-apple-darwin/debug/build/libfuzzer-sys-43b9294388a9aed5/out`
error[E0107]: wrong number of type arguments: expected 2, found 1
  --> fuzz_targets/input_text.rs:15:17
   |
15 |     type Item = Result<InputSeq>;
   |                 ^^^^^^^^^^^^^^^^ expected 2 type arguments

error: aborting due to previous error

For more information about this error, try `rustc --explain E0107`.
error: could not compile `kiro-editor-fuzz`.

Caused by:
  process didn't exit successfully: `rustc --edition=2018 --crate-name input_text fuzz_targets/input_text.rs --color always --crate-type bin --emit=dep-info,link -C debuginfo=2 -C metadata=1f46cebbe29cadde -C extra-filename=-1f46cebbe29cadde --out-dir /Users/rhysd/Develop/github.com/rhysd/kiro-editor/fuzz/target/x86_64-apple-darwin/debug/deps --target x86_64-apple-darwin -C incremental=/Users/rhysd/Develop/github.com/rhysd/kiro-editor/fuzz/target/x86_64-apple-darwin/debug/incremental -L dependency=/Users/rhysd/Develop/github.com/rhysd/kiro-editor/fuzz/target/x86_64-apple-darwin/debug/deps -L dependency=/Users/rhysd/Develop/github.com/rhysd/kiro-editor/fuzz/target/debug/deps --extern kiro_editor=/Users/rhysd/Develop/github.com/rhysd/kiro-editor/fuzz/target/x86_64-apple-darwin/debug/deps/libkiro_editor-06bdf11b37286e1e.rlib --extern libfuzzer_sys=/Users/rhysd/Develop/github.com/rhysd/kiro-editor/fuzz/target/x86_64-apple-darwin/debug/deps/liblibfuzzer_sys-0d3a1916dd5a6e3c.rlib --cfg fuzzing -Cpasses=sancov -Cllvm-args=-sanitizer-coverage-level=4 -Cllvm-args=-sanitizer-coverage-trace-compares -Cllvm-args=-sanitizer-coverage-inline-8bit-counters -Cllvm-args=-sanitizer-coverage-trace-geps -Cllvm-args=-sanitizer-coverage-prune-blocks=0 -Cllvm-args=-sanitizer-coverage-pc-table -Zsanitizer=address -L native=/Users/rhysd/Develop/github.com/rhysd/kiro-editor/fuzz/target/x86_64-apple-darwin/debug/build/jemalloc-sys-470dba03bcb05edc/out/build/lib -L native=/Users/rhysd/Develop/github.com/rhysd/kiro-editor/fuzz/target/x86_64-apple-darwin/debug/build/libfuzzer-sys-43b9294388a9aed5/out` (exit code: 1)
error: could not build fuzz script: "cargo" "build" "--manifest-path" "/Users/rhysd/Develop/github.com/rhysd/kiro-editor/fuzz/Cargo.toml" "--verbose" "--bin" "input_text" "--target" "x86_64-apple-darwin"% 
</code></pre></details>

I think we should replace `eprint!` in `for` loop with `eprintln!`